main.go

```diff
@@ -358,6 +380,22 @@ func main() {
 		}
 	}()
 
+	// tracer sink
+	tracer, err := tracer.GetSink(cfg.Controller.Tracer)
+	defer tracer.Stop()
+
+	if err != nil {
+		logger.Errorw("error while creating tracer sink", "error", err)
+	}
```

Comment:
You can't expect the tracer to be initialized on error, at least usual defer pattern do not defer if error
```suggestion
	if err != nil {
		logger.Errorw("error while creating tracer sink", "error", err)
	} else {
		defer tracer.Stop()
	}
```

Reply:
fixed !